content/sensu-go/6.9/operations/manage-secrets/secrets-providers.md

```diff
@@ -274,11 +355,157 @@ name: vault
 {{< /code >}}
 {{< /language-toggle >}}
 
-### Spec attributes
+### `CyberArkProvider` spec attributes
+
+client       | 
+-------------|------ 
+description  | Map that includes [CyberArkProvider client attributes][20].
+required     | true
+type         | Map of key-value pairs
+example      | {{< language-toggle >}}
+{{< code yml >}}
+client:
+  account: sensu.io
+  appliance_url: http://localhost:8480
+  login: host/Sensu/sensuBackend
+  api_key: CONJUR_API_KEY
+  timeout: 1s
+  tls:
+    ca_cert: "/etc/ssl/certs/conjur_ca_cert.pem"
+  ttl: 60s
+{{< /code >}}
+{{< code json >}}
+{
+  "client": {
+    "account": "sensu.io",
+    "appliance_url": "http://localhost:8480",
+    "login": "host/Sensu/sensuBackend",
+    "api_key": "CONJUR_API_KEY",
+    "timeout": "1s",
+    "tls": {
+      "ca_cert": "/etc/ssl/certs/conjur_ca_cert.pem"
+    },
+    "ttl": "60s"
+  }
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+#### `CyberArkProvider` client attributes
+
+account      | 
+-------------|------ 
+description  | Conjur account name.
+required     | true
+type         | String
+example      | {{< language-toggle >}}
+{{< code yml >}}
+account: sensu.io
+{{< /code >}}
+{{< code json >}}
+{
+  "account": "sensu.io"
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+appliance_url | 
+--------------|------ 
+description   | Conjur appliance URL.
```

Comment:
Where do users get the appliance_url?

Reply:
I chose "appliance url" to match the terminology I had seen in cyberark's docs and their clients. It is simply the http(s) endpoint cyberark is listening on.

If they are using CyberArk Enterprise (their SaaS offering) I assume the Appliance URL would be very clear in some web UI, but I've never seen that product. Otherwise, running a self hosted cluster one would "just know" ™️. e.g. if Sensu Inc. had had one I could imagine it being made available at `https://secrets.sensu.io`